packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -232,12 +232,28 @@ class CameraValue {
 /// To show the camera preview on the screen use a [CameraPreview] widget.
 class CameraController extends ValueNotifier<CameraValue> {
   /// Creates a new camera controller in an uninitialized state.
+  ///
+  /// Deprecated. Please use [withSettings].
```

Comment:
nit: I don't think we necessarily need to deprecate this constructor. I don't see a reason we can't have both. This could still be used by those that don't want to mess with the bitrate, etc....